Johay90.ShiftLogger/ShiftLoggerUI/UI/UserInputManager.cs

```diff
@@ -0,0 +1,134 @@
+﻿using SharedLibrary.DTOs;
+using ShiftLoggerUI.Enums;
+using Spectre.Console;
+
+namespace ShiftLoggerUI.UI
+{
+    internal static class UserInputManager
+    {
+        public static MenuOptions GetMenuOption()
+        {
+            return AnsiConsole.Prompt(new SelectionPrompt<MenuOptions>()
+                .Title("Please choose an [green]API call[/]?")
+                .AddChoices(Enum.GetValues<MenuOptions>())
+                .PageSize(15));
+        }
+
+        public static int GetId() => AnsiConsole.Ask<int>("Type an ID:");
+
+        public static bool Retry()
+        {
+            return AnsiConsole.Ask<bool>("Would you like to try again <True/False?");
+        }
+
+        private static (string name, DateTime dob, string number, string email) CollectEmployeeInfo()
+        {
+            var name = AnsiConsole.Ask<string>("Name:");
+            var date = GetDOB();
+            var dob = new DateTime(day: date.Day, month: date.Month, year: date.Year);
+            var number = AnsiConsole.Ask<string>("Phone Number:");
+            var email = AnsiConsole.Ask<string>("E-Mail:");
+
+            return (name, dob, number, email);
+        }
+
+        public static CreateEmployeeDto CreateEmployee()
+        {
+            var (name, dob, number, email) = CollectEmployeeInfo();
+
+            return new CreateEmployeeDto
+            {
+                Name = name,
+                DateOfBirth = dob,
+                PhoneNumber = number,
+                EmailAddress = email
+            };
+        }
+
+        public static UpdateEmployeeDto UpdateEmployee()
+        {
+            var (name, dob, number, email) = CollectEmployeeInfo();
+
+            return new UpdateEmployeeDto
+            {
+                Name = name,
+                DateOfBirth = dob,
+                PhoneNumber = number,
+                EmailAddress = email
+            };
+        }
+
+        private static (DateTime StartTime, DateTime EndTime) CollectShiftInfo()
+        {
+            DateTime startTime, endTime;
+            TimeSpan shiftLength;
+            bool isValid;
+
+            do
+            {
+                startTime = AnsiConsole.Ask<DateTime>("Start Time? (mm-dd-yyyy HH:mm:ss)");
```

Comment:
The datetime is validating, but there is a small bug.
Now the user can enter dates without time, therefore default value is entered to fill the time part.
But in this application we need to expect the user to enter date and time.
Kindly create a custom validation to insure that the user entered the both date and time.

Reply:
Pushed, now requires the exact "MM-dd-yyyy HH:mm:ss" format for shift times before processing. 

Thank you!